tests/settings_tests/tests.py

```diff
@@ -616,3 +617,21 @@ def test_add_script_name_prefix(self):
                             self.assertEqual(getattr(settings, setting), expected_path)
                         finally:
                             clear_script_prefix()
+
+
+class FromEnvTest(SimpleTestCase):
+    def test_boolable(self):
+        self.assertTrue(boolable("true"))
+        self.assertFalse(boolable("false"))
+        self.assertFalse(boolable("banana"))
```

Comment:
Use `assertIs` to check booleans:

More details as: https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/